### DISCUSSION TITLE
3D Print files

### DISCUSSION THREAD

**Discussion by liamamiller** [CircuitSetup/Time-Circuits-Display#26] [Q&A] [answered]
How do the faceplates attach to the other parts when 3D printing the enclosure files? I see no screw holes or mounting points.

**Comment by Ricky3Nut** [comment, marked as answer]
Double sided scotch tape. Its listed in the supplies. Might want to try something a little stonger, mine tend to peel up slightly. 

⁣Get BlueMail for Android ​
